ci/build-linux-repositories@.service

```
@@ -0,0 +1,21 @@
+# Create separate services with instance names being set to "$server_environment-$repository".
+# See the build-linux-repositories.sh script for details.
+#
+# * Log in to the user with `sudo machinectl shell build@` to get a proper shell
+# * Copy this file to ~/.config/systemd/user/
+# * Run `systemctl --user enable`.... TODO
+#
+# For example:
+# * build-linux-repositories@production-stable.service
+# * build-linux-repositories@staging-beta.service
+# * build-linux-repositories@dev-beta.service
+
+[Unit]
+Description=Mullvad Linux repository generation and upload service
+
+[Service]
+Type=oneshot
+ExecStart=./build-linux-repositories.sh %i
```

```suggestion
EnvironmentFile=/some/path/%i.env
ExecStart=./build-linux-repositories.sh %i
```
IF you want to, you could skip the `build-linux-repositories-config.sh` script and instead have separate environment-config files which get pulled in by the service.

---

Sounds cool! But only two out of six configure variables are dependent on the environment. The rest are the same. So would be lots of duplication potentially. We could use *both* systems. But given how quickly we want to deploy this I'll leave it as is for now. Thanks for the recommendation.